config/from.go

```diff
@@ -42,6 +42,9 @@ func WildcardToRegex(wildcard string) string {
 		wildcard = wildcard[idx+1:]
 	}
 	b.WriteString(regexp.QuoteMeta(wildcard))
+	if stripPort && !strings.Contains(wildcard, ":") {
```

Comment:
I'm trying to think if there are any corner cases here... do we allow IP addresses in `from` URLs? I'm wondering if we should do anything about IPv6 addresses, which could contain `:` characters without specifying a port number.

(Not sure if it's something we want to support though.)

Reply:
good catch